aten/src/ATen/native/native_functions.yaml

```diff
@@ -1153,7 +1165,7 @@
 - func: max(Tensor self, int dim, bool keepdim=False) -> (Tensor values, Tensor indices)
   variants: function, method
 
-- func: max(Tensor self, int dim, bool keepdim=False, *, Tensor(a!) max, Tensor(b!) max_values) -> (Tensor(a!) values, Tensor(b!) indices)
+- func: max(Tensor self, int dim, bool keepdim=False, *, Tensor(a!) max, Tensor(b!) max_indices) -> (Tensor(a!) values, Tensor(b!) indices)
```

Comment:
when did this parameter name change happen?  Is this BC breaking?

Reply:
I think it's a typo, but I'll leave it out of this PR